spec/features/user_can_view_past_rentals_spec.rb

```diff
@@ -0,0 +1,27 @@
+# Background: An existing user that has multiple rentals
+#       As an Authenticated User
+#       When I visit "/rentals"
+#       Then I should see all rentals belonging to me
+
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
+  create_category
+  create_brand
+  create_items
+  create_user
+  create_rental
+
+
+  feature 'viewing past rentals' do
+    it 'shows list of past rentals' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.